core/src/DocTestDataUnicode.hs

```diff
@@ -0,0 +1,18 @@
+{- $setup
+>>> :m
+>>> :set -XQuasiQuotes
+>>> :set -XMagicHash
+>>> :set -XOverloadedStrings
+
+>>> import Streamly.Internal.Unicode.Stream
+>>> import Streamly.Internal.Unicode.String
+
+>>> import qualified Streamly.Data.Fold as Fold
+>>> import qualified Streamly.Data.Stream as Stream
+>>> import qualified Streamly.Internal.Unicode.Array as Array
+>>> import qualified Streamly.Internal.Unicode.Stream as Unicode
+>>> import qualified Streamly.Unicode.Parser as Unicode
```

Comment:
Separate non-internal imports from internal ones. Add a comment before internal imports - see Streamly.Data.Stream.

Reply:
Do this in other modules as well.